### PR TITLE
fix: deprecate the `remote_service_type` in llm model

### PR DESCRIPTION
**Pull request by ashleyxuu**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-bigquery-dataframes/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

**Review by milkshakeiii — APPROVED** [review]

File: bigframes/ml/llm.py [hunk 1]
```diff
@@ -90,7 +90,7 @@ def _create_bqml_model(self):
         )
         if self.model_name == "text-bison":
             options = {
-                "remote_service_type": _REMOTE_TEXT_GENERATOR_MODEL_CODE,
+                "endpoint": _REMOTE_TEXT_GENERATOR_MODEL_CODE,
             }
         else:
```
Comment (GarrettWu) [inline bigframes/ml/llm.py:95]: Don't just call else for all other inputs.

if:
elif:
else:
  raise ValueError

File: bigframes/ml/llm.py [hunk 2]
```diff
@@ -24,11 +24,11 @@
 from bigframes.ml import base, core, globals, utils
 import bigframes.pandas as bpd
 
-_REMOTE_TEXT_GENERATOR_MODEL_CODE = "CLOUD_AI_LARGE_LANGUAGE_MODEL_V1"
+_REMOTE_TEXT_GENERATOR_MODEL_CODE = "text-bison"
```
[the remark this answers is not in the document]
Comment (GarrettWu) [reply bigframes/ml/llm.py:27]: Also are they always the same as the param "model_name" of the models? Then we may want to merge them.

[the remark this answers is not in the document]
Comment (ashleyxuu) [reply bigframes/ml/llm.py]: Done

File: bigframes/ml/llm.py (hunk 1 above)
Comment (ashleyxuu) [reply bigframes/ml/llm.py]: Done

**Review by GarrettWu — APPROVED** [review]

**Pull request merged**